config/kernel/linux-rk3399-legacy.config

```diff
@@ -5887,7 +5893,7 @@ CONFIG_RK_FLASH=m
 CONFIG_ARM_PSCI_FW=y
 # CONFIG_FIRMWARE_MEMMAP is not set
 CONFIG_HAVE_ARM_SMCCC=y
-# CONFIG_ROCKCHIP_SIP is not set
+CONFIG_ROCKCHIP_SIP=y
 # CONFIG_ACPI is not set
 
 #
```

Comment:
We are in the last test phase for new release. I'll check OPi 4 and if it doesn't break also on Nanopi M4, we can move this in.

Reply:
Both boots.